traildb/traildb.py

```diff
@@ -224,6 +229,15 @@ def __del__(self):
     def __iter__(self):
         return self
 
+    def get_trail(self, trail_id):
+        if lib.tdb_get_trail(self.cursor, trail_id) != 0:
+            raise TrailDBError("Failed to initalize trail in cursor")
+
+            if self.event_filter_obj:
+                if lib.tdb_cursor_set_event_filter(self.cursor, event_filter_obj.flt):
+                    raise TrailDBError("cursor_set_event_filter failed")
```

Comment:
This indentation looks wrong. How is it possible to reach this code immediately after raising an exception? @knutin

Reply:
It's probably just misindented. I'll pull that indentation back a bit.